packages/twenty-server/src/engine/api/graphql/workspace-query-runner/workspace-query-hook/interfaces/workspace-query-hook.interface.ts

```diff
@@ -1,19 +1,20 @@
+import { QueryResultFieldValue } from 'src/engine/api/graphql/workspace-query-runner/factories/query-result-getters/interfaces/query-result-field-value';
 import { ResolverArgs } from 'src/engine/api/graphql/workspace-resolver-builder/interfaces/workspace-resolvers-builder.interface';
 
 import { AuthContext } from 'src/engine/core-modules/auth/types/auth-context.type';
 
-export interface WorkspaceQueryHookInstance {
+export interface WorkspacePreQueryHookInstance {
   execute(
     authContext: AuthContext,
     objectName: string,
     payload: ResolverArgs,
   ): Promise<ResolverArgs>;
 }
 
-export interface WorkspaceQueryPostHookInstance {
+export interface WorkspacePostQueryHookInstance {
   execute(
     authContext: AuthContext,
     objectName: string,
-    payload: unknown[],
+    payload: QueryResultFieldValue,
```

Comment:
this was not true: depending if it's a find-many or a find-one or a create-many post query hook